keyboards/0xcb/splaytoraid/keymaps/pi/keymap.c

```diff
@@ -0,0 +1,350 @@
+    _______,  _______,  _______,  _______,  _______,                       _______,  _______,  _______,  _______,  _______,
+    _______,  _______,  _______,  _______,  _______,                       _______,  _______,  _______,  _______,  _______,
+                        _______,  _______,  _______,       _______,        _______,  _______,  _______
+ )*/
+};
+
+// ┌────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┐
+// │ D I E S  U N D  D A S                                                                                                                      │
+// └────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┘
+// ▝▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▘
+bool caps_word_press_user(uint16_t keycode) {
+  switch (keycode) {
+    // Keycodes that continue Caps Word, with shift applied.
+    case KC_A ... KC_Z:
+      add_weak_mods(MOD_BIT(KC_LSFT));  // Apply shift to the next key.
+      return true;
+
+    // Keycodes that continue Caps Word, without shifting.
+    case KC_1 ... KC_0:
+    case KC_BSPC:
+    case KC_DEL:
+    case KC_MINS:
+    case KC_UNDS:
+    case KC_LSFT:
+    case KC_RSFT:
+      return true;
+
+    default:
+      return false;  // Deactivate Caps Word.
+  }
+}
+
+// ┌────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┐
+// │ M A C R O S                                                                                                                                │
+// └────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┘
+// ▝▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▘
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+    switch (keycode) {
+
+        case OS_SWAP:
+            if (record->event.pressed) {
+                if (!keymap_config.swap_lctl_lgui) {
+                  keymap_config.swap_lctl_lgui = true;  // ─── MAC
+                  #ifdef AUDIO_ENABLE
+                    PLAY_SONG(mac_song);
+                  #endif // AUDIO_ENABLE
+                }
+                else {
+                  keymap_config.swap_lctl_lgui = false; // ─── WIN
+                  #ifdef AUDIO_ENABLE
+                    PLAY_SONG(winxp_song);
+                  #endif // AUDIO_ENABLE
+                }
+            eeconfig_update_keymap(keymap_config.raw);
+            clear_keyboard();  // ──── clear to prevent stuck keys
+            return false;
+          }
+
+
```

Comment:
[magic](https://docs.qmk.fm/#/keycodes_magic) keycodes should be used here instead, as it seems this keyboard does not have audio support, and this keymap was entirely copy/pasted from another keyboard.
```suggestion
```